src/stream.ts

```diff
@@ -1,13 +1,15 @@
-import { Stream, StreamStat, Direction } from '@libp2p/interface-connection';
-import { Source } from 'it-stream-types';
-import { Sink } from 'it-stream-types';
-import { pushable, Pushable } from 'it-pushable';
-import defer, { DeferredPromise } from 'p-defer';
+import {Stream, StreamStat, Direction} from '@libp2p/interface-connection';
+import {Source} from 'it-stream-types';
+import {Sink} from 'it-stream-types';
+import {pushable} from 'it-pushable';
+import * as lp from 'it-length-prefixed';
+import { pipe } from 'it-pipe';
+import defer, {DeferredPromise} from 'p-defer';
 import merge from 'it-merge';
-import { Uint8ArrayList } from 'uint8arraylist';
-import { fromString } from 'uint8arrays/from-string';
-import { logger } from '@libp2p/logger';
+import {Uint8ArrayList} from 'uint8arraylist';
+import {logger} from '@libp2p/logger';
```

Comment:
least important comment of all, but why are we something importing as `{ x }` and sometimes as `{x}`? Super unimportant, but that's what a linter is for, no?